<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTip">
   <string>PSA1_PRO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BTN_PSA1_IN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_IN</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA1_PRO">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_PRO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA1_BAL">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_BAL</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA1_CLR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_CLR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA2_IN">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_IN</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA2_BAL">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_BAL</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA2_CLR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA2_CLR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PSA2_PRO">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PSA1_PRO</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>20</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>20</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PROD_OUT">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PROD_OUT</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PROD_PSA1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PROD_PSA1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PROD_PSA2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>PROD_PSA2</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>20</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>20</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>20</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PSA 1</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PSA 2</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PROD</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_COL2_IN">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>COL2_IN</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>resource/valveoff.png</normaloff>
      <normalon>:/logo_valve/resource/valveon.png</normalon>resource/valveoff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>line_13</zorder>
   <zorder>line_12</zorder>
   <zorder>line_11</zorder>
   <zorder>line_9</zorder>
   <zorder>line_6</zorder>
   <zorder>line_4</zorder>
   <zorder>line</zorder>
   <zorder>BTN_PSA1_PRO</zorder>
   <zorder>BTN_PSA1_CLR</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>BTN_PSA1_BAL</zorder>
   <zorder>BTN_PSA1_IN</zorder>
   <zorder>BTN_PSA2_IN</zorder>
   <zorder>BTN_PSA2_BAL</zorder>
   <zorder>BTN_PSA2_CLR</zorder>
   <zorder>BTN_PSA2_PRO</zorder>
   <zorder>line_5</zorder>
   <zorder>graphicsView</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>line_10</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>BTN_PROD_OUT</zorder>
   <zorder>BTN_PROD_PSA1</zorder>
   <zorder>BTN_PROD_PSA2</zorder>
   <zorder>line_14</zorder>
   <zorder>line_15</zorder>
   <zorder>line_16</zorder>
   <zorder>line_17</zorder>
   <zorder>line_18</zorder>
   <zorder>line_19</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>BTN_COL2_IN</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QMenu" name="menuAuto">
     <property name="title">
      <string>Auto</string>
     </property>
     <addaction name="actionStart"/>
     <addaction name="actionRepeat"/>
    </widget>
    <addaction name="menuAuto"/>
    <addaction name="actionManual"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
